core/lib/dal/migrations/20240401105036_block-gas-limit.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE miniblocks ADD COLUMN gas_limit BIGINT;
```

Comment:
BIGINT is `i64`, it won't fit `u64::MAX`, you should use `NUMERIC`

Reply:
while we are here, should we move to u256?

Reply:
i64 is enough for our purposes, in reality the block gas limit we'll use is 2^50, which fits into i64

Reply:
>while we are here, should we move to u256?

I am fine with that, especially since we anyway use `U256` for the `Fee` struct, so if you are fine with migrating to numeric, I am fine also, but you can better assess the impact on performance